_posts/2016-02-06-office-air-printer.md

```diff
@@ -0,0 +1,42 @@
+---
+layout: post
+title: Copiers and printers puff out particulates
+author: neuroamanda
+---
+
+_Office devices vent some unwelcome chemicals_
+
+Office air can contain many chemical compounds that come from sources like carpet cleaning, furniture, dry cleaned clothes, and office devices.
+These can cause allergic reactions, respiratory illness, headaches, and other symptoms collectively called [sick building syndrome](https://en.wikipedia.org/wiki/Sick_building_syndrome).
+[A recent study](http://dx.doi.org/10.1007/s11356-014-3672-3) specifically looked at the amounts and types of chemicals released by printers and copiers.
+
+When ink and toner are heated during the printing process, trace amounts of their constituent chemicals are released into the surrounding air.
+The amounts depend on the temperature and the frequency of use and operation of the devices.
+In addition, the circuit boards and plastic parts of devices can give off their own emissions.
+Fine aerosol paper particles can also contribute to the office air cocktail of volatile organic compounds (VOCs).
+Volatile refers to the fact that a large number of molecules are released, not that these compounds are inherently dangerous or artificial, though one man-made VOC is probably very familiar: [new car smell](https://en.wikipedia.org/wiki/New_car_smell).
+
+![Exploded toner cartridges]({{ site.url }}/images/2016-02-06-office-air-printer.jpg)
+
+> _Exploded toner cartridges. [Image](https://www.flickr.com/photos/makenosound/2557526304/) by Max Wheeler / [CC BY-NC-ND 2.0](https://creativecommons.org/licenses/by-nc-nd/2.0/)_
+
+Researchers tested seven different inkjet and laser printers and copiers inside a Plexiglas chamber to capture any released substances.
+The devices were all manufactured during the 1990s and 2000s, but their manufacturers and models were not identified.
+Air inside the chambers was sampled for five hours a day over four days, with printing taking place for 30 minutes at a time followed by a 30-minute break.
+To calculate how much each device was emitting, the researchers took into account the air flow inside the chamber, its volume, the sampling time and volumes they collected, the printing/copying time, and other factors.
+
+Over 60 compounds were identified in the air samples, including toluene, benzene, xylenes, ethylbenzene and styrene which, according to the authors "pose [a] hazard to human health when present in air."
```

Comment:
Broken commas, dude. Comma needed before "which" and after "authors". Also, it's not a complete quote, so the full stop should go outside the quotation mark, but I know your style is different (read: wrong). :P